charts/oes/values.yaml

```diff
@@ -711,6 +703,17 @@ kibana:
 # Spinnaker instance configuration
 #####################################################
 spinnaker:
+  installOpenLdap: true
```

Comment:
installOpenLdap is already there in line no. 580. Let's not have duplicates. If you want keep it under global section that will be there at the beginning of this file so that we can also use it under spinnaker sub chart